nixos/modules/services/display-managers/sddm.nix

```diff
@@ -77,12 +84,6 @@ let
       DisplayStopCommand = toString Xstop;
       EnableHiDPI = cfg.enableHidpi;
     };
-
-    Wayland = {
-      EnableHiDPI = cfg.enableHidpi;
-      SessionDir = "${dmcfg.sessionData.desktops}/share/wayland-sessions";
-      CompositorCommand = lib.optionalString cfg.wayland.enable cfg.wayland.compositorCommand;
-    };
   } // optionalAttrs dmcfg.autoLogin.enable {
```

Comment:
We should probably sort this after autologin

Reply:
i am not sure if i understand you here. Do you mean here the wayland node, as the comment is on the autoLogin node, or which exactly you want me to move?